cuda_core/tests/test_module.py

```diff
@@ -100,6 +111,13 @@ def test_object_code_load_cubin(get_saxpy_kernel):
     mod.get_kernel("saxpy<double>")  # force loading
 
 
+def test_object_code_load_ptx(get_saxpy_kernel_ptx):
+    ptx, mod = get_saxpy_kernel_ptx
+    sym_map = mod._sym_map
+    mod = ObjectCode.from_ptx(ptx, symbol_mapping=sym_map)
+    mod.get_kernel("saxpy<double>")
```

Comment:
All minor points:

I find the `# force loading` comment (like above in the existing code) useful.

I generally avoid reusing variable names, i.e. I'd look for another name for the 2nd `mod`, or even better give both `mods` another name to distinguish them and make this easier to read.

Is there something easy that we could `assert` here? It's unusual to just run code without any asserts.

Reply:
I updated the names and added an assert that verifies roundtripping the ptx here.